classes/classes/Output.as

```diff
@@ -39,14 +39,7 @@ package classes
 		/**
 		 * Passes the text through the parser and adds it to the output-buffer 
 		 *
-		 * In debug-mode the output is directly flushed to the GUI.
 		 * If you want to clear the output before adding text, use clear(true) or just clear()
-		 * The old second param `purgeText:Boolean = false` from outputText(...) is not supported anymore
-		 * in favor of using clear() and will never return.
-		 * 
-		 * Unfortunately no one succeded to support markdown formatting for CoC, although this was attempted.
-		 *
-		 * This must not be made possible to be called directly from the outside, use wrapper-methods instead.
```

Comment:
Mentioning stuff that was deprecated and removed over a year ago is useless clutter now.